packages/pwa-kit-dev/src/ssr/server/build-dev-server.js

```diff
@@ -143,6 +143,18 @@ export const DevServerMixin = {
                 dotFiles: 'deny'
             })
         )
+
+        app.get('/__mrt/clear-browser-data', (_, res) => {
+            console.log(
+                chalk.cyan('Clearing browser data'),
+                '(cache, service worker, web storage for browsers supporting Clear-Site-Data header)'
```

Comment:
Might be helpful to link to a support table like
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Clear-Site-Data#browser_compatibility. (You link to MDN in a code comment, but that's not visible to developers seeing this message in the console.)